features/staging_commands.feature

```diff
@@ -350,6 +350,7 @@ Feature: Enable command behaviour when attached to an UA staging subscription
             Updating package lists
             Installing CIS Audit packages
             CIS Audit enabled
+            Visit https://ubuntu.com/security/cis to learn how to use the CIS packages
```

Comment:
@mitechie @blackboxsw I went ahead and changed the message to point to the URL even though it isn't live yet. Let me know what you think of the wording.

Reply:
Now that I see the message, this is not the URI to learn how to use the CIS packages. This is the URI to learn about CIS and Ubuntu. If your intention is to point to technical instructions, then a more appropriate URI is:
https://security-certs.docs.ubuntu.com/en/cis